src/client/components/ChannelHeadNav/ChannelHeadNav.style.css

```diff
@@ -0,0 +1,30 @@
+.channel-head-div {
+  position: relative;
+  height: 10vh;
+  background-color: lightblue;
+  display: flex;
+  justify-items: flex-start;
+  flex-direction: row;
+}
+.backArrowIcon {
```

Comment:
Sometimes for the sake of isolation we prefix all these with the root component `.channel-head-div`. This way in case someone else has a similar name the styles do not conflict.